README.md

```diff
@@ -28,12 +20,26 @@ If you are an existing FarmBot user and just need to upgrade your firmware, cons
  0. Plug your SD Card into your RPi3
  0. Plug your Arduino into your RPi3
  0. Plug your power into your RPi3
- 0. From a WiFi enabled device, search for the SSID `FarmbotConfigurator`
+ 0. From a WiFi enabled device*, search for the SSID `FarmbotConfigurator`
  0. Connect to that and open a web browser to [http://192.168.24.1/](http://192.168.24.1)
- 0. Follow the on screen instruction
+ 0. Follow the on screen instructions to configure your FarmBot. Once you save your configuration FarmBot will connect to your home WiFi network and to the FarmBot web application.
+ 
+Asterisk: If you are using a smartphone you may need to disable cellular data to allow your phone's browser to connect to the configurator.
```

Comment:
`Asterisk` is an odd way to make this connection.
Typically you would use the same sigil.